src/ColliveryApiRequest/ColliveryApiRequest.php

```diff
@@ -30,11 +26,11 @@ public function getHeaders(): array
         return [
             'Content-Type' => 'application/json',
             'Accept' => 'application/json',
-            'X-App-Name' => 'My Custom App',
-            'X-App-Version' => '0.2.1',
-            'X-App-Host' => '.NET Framework 4.8',
+            'X-App-Name' => $this->config['app_name'],
+            'X-App-Version' => $this->config['app_version'],
+            'X-App-Host' => $this->config['app_host'],
             'X-App-Lang' => 'C#',
```

Comment:
App-Lang needs to be PHP but rather from config so version can be set

Reply:
[38651bc]